isatty_windows_test.go

```diff
@@ -23,6 +23,9 @@ func TestCygwinPipeName(t *testing.T) {
 		{`\cygwin-e022582115c10879-pty4-from-master`, true},
 		{`\msys-e022582115c10879-pty4-to-master`, true},
 		{`\cygwin-e022582115c10879-pty4-to-master`, true},
+		{`\Device\NamedPipe\cygwin-e022582115c10879-pty4-from-master`, true},
+		{`Device\NamedPipe\msys-e022582115c10879-pty4-to-master`, true},
+		{`Device\NamedPipe\cygwin-e022582115c10879-pty4-to-master`, true},
```

Comment:
you change is checking prefix string `\Device\...` but this test does not have leading backslash.